src/style.css

```diff
@@ -1,3 +1,71 @@
-body {
+* {
```

Comment:
better use: 

```
html {
  box-sizing: border-box;
}

*,
*::after,
*::before {
  box-sizing: inherit;
}
```

instead of 
```
* {
  box-sizing: inherit;
}
```